### ISSUE TITLE
Debugger dont paused inside one-line lambda(delegate)

### ISSUE THREAD

**Issue by VISTALL** [consulo/consulo-dotnet#59]
```csharp
using System;
using System.Linq;
using System.Collections.Generic;

public class Program {

    public static void Main() {
        var list = new List<string>();
        list.Add("test1234");
        list.Add("test1");
        list.Add("test142412");
        list.Add("test4124");

        list.Sort((x, y) => x.CompareTo(y));  // debugger dont works for lambda

        list.Sort(delegate(String x, String y) { return x.CompareTo(y); });    // debugger dont works for delegate

        list.Sort(delegate(String x, String y) {
            return x.CompareTo(y);   // debugger works on delegate
        });
    }
}

```

**Comment by VISTALL** [comment]
fixed by a94849b00ae3589e0c01441b05e5005905ee11d0

**Comment by VISTALL** [comment]
https://github.com/consulo/consulo-dotnet/commit/a94849b00ae3589e0c01441b05e5005905ee11d0
